docs/handbook/systems-administration.md

```diff
@@ -81,24 +94,114 @@ su
+permits select root access on a user, command or machine level. It effectively
+allows users to execute a number of commands as another user. This other user is
+most frequently root.
 
-To enable a user the ability to use `sudo`, the superuser edits `/etc/sudoers`.
-This should ideally be done as follows:
+
+#### Configuring sudo
+
+To enable a user the ability to use sudo, `/etc/sudoers` must be edited.
+It can be configured such that a user is privileged to use one, several or all
+superuser commands. Syntax errors to this file can cause severe havoc to the
+system. To this end this file should ideally be edited using `visudo`. This
+performs various syntax checks. `sudoers`(4) provides details on the precise means
+to appropriately add a user to use sudo.
+
+visudo will invoke the editor defined in the system environment variable EDITOR,
+and defaults to vi. If you'd prefer to use another editor, for example emacs,
+call visudo as follows: `EDITOR=emacs visudo`
+
+More flexible, however, is assigning one or several commands to a group of
+users. Users can then be added to this group. The syntax of entries in
+`/etc/sudoers` is as follows:
 
 ```
-visudo
+user hostlist=(userlist) commandlist
+%group hostlist=(userlist) commandlist
 ```
 
-This performs various syntax checks.
-sudoers(1) provides details on the precise
-means to appropriately add a user to use sudo.
+So, for example, root would typically have the following entry: `root ALL =
+(ALL) ALL`. A group is prefixed by a '%'.
 
-Example:
+
+*Example: Assigning several commands to a single user*
+<div class="well">
+Permit a user to run the lpadmin and lprm commands
+
+To allow user 'whoever' the ability to configure CUPS (Common UNIX Printing
+System) using the `lpadmin` command, the following entry suffices:
+
+```
+whoever ALL=NOPASSWD:/usr/sbin/lpadmin, /usr/sbin/lprm
+```
+
+The `NOPASSWD` parameter allows the user to issue the command without having to
+enter a password.
+</div>
+
+Instead of a user, we can assign a group, so that anyone belonging to the group
+has access to the relevant commands assigned to the group.
+
+
+#### Tasks, Groups and Users
+
+<!--Configuring sudo for multiple commands for multiple users-->
+
+The above example is a simple mechanism in which an account is assigned one, or
+more administrative commands in which to carry out administrative
+duties. However, this does not scale particularly well. On a system with a small
+number of users this might be sufficient.  On a system with several thousand
+users another mechanism is required.
+
+Instead of assigning one or more commands to a single user, it is more versatile
+to consider administrative duties as a set of tasks, whereby a task can be
+viewed upon as a set of related commands required to perform the task. We can
+then assign the task to one or more users.
+
+This can be implemented as follows:
+
+- For each task, create a group (more about groups later)
```

Comment:
Where? We reference to group description, but do not provide it. Likely, have to remove reference.